checkapi/internal/ast.go

```diff
@@ -148,9 +180,6 @@ func readFile(ignoredFunctions []string, f *ast.File, result *API) {
 			}
 		}
 		if fn, isFn := d.(*ast.FuncDecl); isFn {
-			if !fn.Name.IsExported() {
```

Comment:
Any reason why this was removed?

Reply:
we only captured functions that were exported, but with this change we need to capture unexported functions such as `createDefaultConfig` so we can source the Config struct.